Dolphin/windWakerInterface.py

```diff
@@ -1,57 +1,57 @@
 import random
-
 import dolphin_memory_engine as dme
-
+from typing import List
 import Dolphin.windWakerResources as WWR
 from Model.config import Config
 
 random_rupoors = Config.get_config().Random_Rupoors
 
-def hook():
+
+def hook() -> None:
     dme.hook()
 
 
-def assert_hook():
+def assert_hook() -> None:
```

Comment:
This can raise an Exception, I don't know if Typing handles that (For example how Java has `throws Exception` after a function declaration.)

Reply:
Not possible with python unfortunately:
https://stackoverflow.com/questions/44282268/python-type-hinting-with-exceptions